2024/11/4/cancellable-interval/cancellable-interval.ts

```
@@ -0,0 +1,10 @@
+export default function setCancellableInterval(
+  callback: Function,
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)